samples/GalleryApp/SkillPageBase.cs

```diff
@@ -177,5 +206,24 @@ protected async Task UpdateMediaSourceButtonsAsync(bool enableButtons)
                 await Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, async () => await UpdateMediaSourceButtonsAsync(enableButtons));
             }
         }
+
+        /// <summary>
+        /// Update skill status indicators' colors (inside the Status box)
+        /// </summary>
+        /// <param name="newBindSkillIndicatorColor"></param>
+        /// <param name="newEvaluateSkillIndicatorColor"></param>
+        /// <returns></returns>
+        protected async Task UpdateIndicators(SolidColorBrush newBindSkillIndicatorColor, SolidColorBrush newEvaluateSkillIndicatorColor)
```

Comment:
>SolidColorBrush [](start = 46, length = 15)

nit: This method sometime is used to update only one of the indicator, if you could offer an option to only pass 1 of the color at a time in these case, you would sae a UI update. (i.e. allow null to be passed whcih bypasses the update, or check if the current value is already set and bypass, etc.)

Reply:
Could not figure out how to pass the xaml object or the reference of the member attribute as a parameter to the method. So the current approach is that all the indicators' colors are stored inside IndicatorColorList.

---
In reply to: [308045508](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/49#discussion_r308045508) [](ancestors = 308045508)

Reply:
something you could do as well is use an enum that defines states instead of static colors. Then you would have a mapping between these states and the colors you wish to use, in the advent you want to tweak them without altering all the files where you pass a color directly (i.e. UpdateIndicator(UpdateSteps.Binding); )

---
In reply to: [308486067](https://github.com/microsoft/WindowsVisionSkillsPreview/pull/49#discussion_r308486067) [](ancestors = 308486067,308045508)